pydicom/tests/test_multival.py

```diff
@@ -129,6 +132,17 @@ def test_not_equal(self):
         self.assertFalse(multival != multival2)
         self.assertTrue(multival != multival3)
 
+    def test_str_rep(self):
+        """MultiValue: test print output"""
+        multival = MultiValue(IS, [])
+        self.assertEqual('', str(multival))
+        multival = MultiValue(compat.text_type, [1, 2, 3])
+        self.assertEqual("['1', '2', '3']", str(multival))
+        multival = MultiValue(int, [1, 2, 3])
+        self.assertEqual('[1, 2, 3]', str(multival))
+        multival = MultiValue(float, [1.1, 2.2, 3.3])
+        self.assertEqual('[1.1, 2.2, 3.3]', str(multival))
+
```

Comment:
Do we know that we will only get one decimal place for these on all python versions and platforms?  Just curious, have been stung by float representations many times.  Just in a test, so no big deal anyway.

Reply:
I wasn't sure about this either, but as all tests passed...